services/ui-src/src/components/fields/ChoiceField.tsx

```diff
@@ -5,12 +5,14 @@ import { Choice as CmsdsChoice } from "@cmsgov/design-system";
 import { Box, Text } from "@chakra-ui/react";
 // utils
 import { AnyObject } from "types";
+import { labelTextWithOptional } from "utils";
 
 export const ChoiceField = ({
   name,
   label,
   hint,
   sxOverride,
+  styleAsOptional,
```

Comment:
I'm sure I'm just missing it, but where is 'styleAsOptional' called? is it part of the report json?

Reply:
it's an attribute that gets added in the 'props' object in the mcpar.json

Reply:
It's not getting used yet though so I couldn't add it.

Reply:
It will be in MLR, but not yet in MCPAR.